### DISCUSSION TITLE
DID - production environment

### DISCUSSION THREAD

**Discussion by xdxxxdx** [eclipse-edc/.github#88] [Q&A] [answered]
Hello EDC team,

I am looking at the solution provided by EDC regarding the DID. 
https://github.com/eclipse-edc/Connector/blob/0adb7a47407ff3e80d809361ea38b526b7a4f536/extensions/common/iam/decentralized-identity/README.md?plain=1#L56
mentioned that the repo is not supposed to be used at the production level. 
But in the IDSA report regarding the EDC. 
https://internationaldataspaces.org/wp-content/uploads/dlm_uploads/IDSA-Data-Connector-Report-90-No-10-November-2023-2.pdf
<img width="314" alt="image" src="https://github.com/eclipse-edc/.github/assets/15314807/160df2d3-6c5c-48b6-be70-e1c7ee84f97a">
The maturity level is marked as TRL 8-9, which indicates the code base can be used in the production level.
I am just wondering what is the situation right now.

Thanks

**Comment by ndr-brt** [comment]
the warning is about the module, not about the whole project.

**Reply by xdxxxdx** [reply]
Thanks @ndr-brt , is there a roadmap for making that DID module production level? As it is very critical for identity management nowadays.  :)

**Comment by mspiekermann** [comment, marked as answer]
The roadmap is closely linked to the Eclipse Dataspace Decentralized Claims Protocol [DCP](https://projects.eclipse.org/projects/technology.dataspace-dcp). The EDC will be definitely one of the reference implementations and thereby adopt the specification. We will do implementation work in parallel, so not waiting for the DCP to be finished, but still depend on specification's milage.
